### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).

It also includes following components:

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/admin/messages/messages_ka.properties
Comment (ahus1) [inline themes/src/main/resources-community/theme/base/admin/messages/messages_ka.properties:45]: Could you please check this translation? When using Google translate, it translates to "You cannot set write-only and read-only alms at the same time" where "alms" should probably be "attribute". Maybe it is just an error in Google translate, still I'd like you to double-check. Thanks!

File: themes/src/main/resources-community/theme/base/login/messages/messages_ka.properties
Comment (ahus1) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ka.properties:154]: "Passkey" is an industry term, and I'd prefer not to translate it. I ass this to this message, and it applies to all messages that contain "Passkey".

See https://fidoalliance.org/passkeys/ for more information about Passkey. "Passkeys" (the plural) usually refers to the method of using the standard and the procedure. We use the singular "Passkey" to refer to a single key pair of a user.

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you @NorwayFun for this translation! I did a brief review using Google Translate as much as I could, see below for some comments.

About non-translated terms: The Glossary contains some terms that are not translated. We're still at the beginning of this, and it would need to be agreed across the different languages. For the German translation, the words "Client/Clients" and "Session/Sessions" are not translated, still we might need to have a more in-depth discussion on that and you are free to keep them translated for now until we had that discussion.

File: themes/src/main/resources-community/theme/base/admin/messages/messages_ka.properties
Comment (NorwayFun) [reply themes/src/main/resources-community/theme/base/admin/messages/messages_ka.properties:45]: Well, ალმები - almebi - flags - can be said anything which is either set or not. I can translate it as attribute too, np to me if you prefer so.

Comment (ahus1) [reply themes/src/main/resources-community/theme/base/admin/messages/messages_ka.properties:45]: Just see it as a comment from me, I'll leave that to a native speaker to decide. I see that the English original is a bit vague as well, as it states "Can not set write-only and read-only together", as it should probably read "Can not set write-only and read-only for an attribute". Still, changing the English original is out of scope for this PR.

File: themes/src/main/resources-community/theme/base/login/messages/messages_ka.properties
Comment (NorwayFun) [reply themes/src/main/resources-community/theme/base/login/messages/messages_ka.properties:154]: OK, I replaced every Passkey with latin variant :)

**Review by ahus1 — APPROVED** [review]
Thank you for these changes, they look good to me. I'm ok to merge them once the build is green.

**Pull request merged**
